freetextresponse/freetextresponse.py

```diff
@@ -86,6 +144,23 @@ def workbench_scenarios():
         default=True,
         scope=Scope.settings,
     )
+    display_other_student_responses = Boolean(
+        display_name=_('Display Other Student Responses'),
+        help=_(
+            'This will display other student responses to the '
+            'student after they submit their response.'
+        ),
+        default=False,
+        scope=Scope.settings,
+    )
+    system_answers = List(
+        default=[], scope=Scope.user_state_summary,
+        help=_("System selected answers to give to students"),
+    )
+    other_responses_shown = List(
+        default=[], scope=Scope.user_state,
```

Comment:
Per style guides, if you're splitting _any_ kwargs across multiple lines, you have to split _all_ of them.